src/game.py

```diff
@@ -1,22 +1,58 @@
+from src.hand import *
+
 class Game:
     # init game with deck, discard deck, and player count
-    def __init__(self, player_count)
+    def __init__(self, player_count, game_points):
+        game_deck = Deck(True)
+        discard_deck = Deck()
+        number_players = player_count
+        max_points = game_points
+
+        hands = []
+
+        for i in range(0, player_count):
+            hands.append(Hand(game_deck))
 
     # resets the deck
-    def reset(self):
+    def reset(self, number_players, hands):
+        game_deck = Deck(True)
+        discard_deck = Deck()
 
-    # adds score for a player
+        hands = []
+        for i in range(0, number_players):
+            hands.append(Hand(game_deck))
+
+    # Calculates and adds score for each player
     def add_score(self, player):
+        print("I compile")
```

Comment:
why do we have this method. The scores should be kept by the individual players and the game should just access that member variable to check end game

Reply:
you made this method

Reply:
alex, when I made all this methods, I had a completely different structure for how I wanted to do things. You changed that structure and therefore you have to update things that were previously used that are no longer valuable. It's not simply what I add and what you add. If I wanted to rewrite the structure of something, that includes ommiting previous things under my PR.